llama_index/__init__.py

```diff
@@ -132,11 +132,13 @@
 # vellum
 from llama_index.llm_predictor.vellum import VellumPredictor, VellumPromptRegistry
 
+# import global eval handler
+from llama_index.callbacks.global_eval_handlers import set_global_eval_handler
```

Comment:
why do we call it eval? this is for tracing?

Reply:
Also, I think the `set_global_eval_handler` function is missing 👁️

Reply:
yeah i could just call it "set_global_handler"